test/apps/spmv/csr_perload.cpp

```diff
@@ -28,12 +28,11 @@
  *
  */
 #include "common/fam_options.h"
+#include "common/fam_test_config.h"
```

Comment:
Can you change create_region calls in this file (apps) in place of adding common/fam_test_config.h